src/lib/chat/index.ts

```diff
@@ -203,6 +204,10 @@ export class Chat {
     return this.matrix.cancelAndResendKeyRequests();
   }
 
+  async addMembersToRoom(roomId: string, users: User[]): Promise<void> {
+    return this.client.addMembersToRoom(roomId, users);
```

Comment:
Simpler to just use `this.matrix.addMembersToRoom...` and just not add it to the common interface. We need to remove the interface anyway.

Reply:
Updated

Reply:
You forgot to remove the method from the `IChatClient` interface and the SendbirdClient. That would be the reason to not use `this.client` here...so we don't have to maintain useless empty functionality.